addons/frag/functions/fnc_dev_sphereDraw.sqf

```diff
@@ -0,0 +1,39 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Add a colored sphere at a specified point.
```

Comment:
```suggestion
 * Add a colored sphere at the specified point.
```

Reply:
I think the original answers the question "what does this function do" more clearly.